data/json/effects.json

```diff
@@ -676,6 +676,60 @@
             "sleep_chance": [101]
         }
     },
+    {
+        "type": "effect_type",
+        "id": "bandaged",
+        "name": ["Bandaged"],
+        "desc": ["Your wounds are bandaged."],
+        "main_parts_only": true,
+        "rating": "good",
+        "int_dur_factor": 3600,
+        "max_intensity": 16,
+        "//": "How many intensity levels may apply effects",
+        "max_effective_intensity": 8,
+        "max_duration": 57600,
+        "base_mods": {
+            "//": "Default healing rate (x100000)", 
+            "healing_rate": [20],
+            "//": "Healing rate if bandaged and disinfected (x100000)", 
+            "healing_rate_w_other_effects": [30],
+            "//": "Multiplier for additional healing from regenerative mutations (x10)", 
+            "healing_mult_regen": [10],
+            "//": "Rate multipliers for head and torso (x10)", 
+            "healing_head_mult": [5],
+            "healing_torso_mult": [15],
+            "//": "Rate multiplier if asleep (x10)", 
+            "healing_sleep_mult": [20]
```

Comment:
All those extra multipliers look like a big overkill. Especially the asleep and mutation rate ones.
Would be much better if you calculated those rates as multipliers of base rate, without specifying them in the json.

For example, like this:
* Base healing rate is `sleep_heal`
* Current healing rate (due to mutations) is `cur_sleep_heal`
* Base healing rate from item is `item_heal`
* When awake, heal only `item_heal`, mutations are ignored
* When asleep, heal `item_heal * (1 + cur_sleep_heal / sleep_heal)`